Setup.sh

```diff
@@ -18,6 +21,9 @@ ADD_COMMAND_TO_HOOK() {
 
   if [ ! -f "$HOOK_FILE" ]; then
     touch "$HOOK_FILE"
+    echo -e "#!/usr/bin/env bash\n" > "$HOOK_FILE"
+    # https://stackoverflow.com/questions/3417896/how-do-i-prompt-the-user-from-within-a-commit-msg-hook
+    echo "exec < /dev/tty" >> "$HOOK_FILE"
```

Comment:
Allow user input from the rest of the hook script. Without this, `read` commands could not get user input.